ql/src/java/org/apache/hadoop/hive/ql/parse/TezCompiler.java

```diff
@@ -991,35 +987,6 @@ private void removeSemiJoinIfNoStats(OptimizeTezProcContext procCtx)
     ogw.startWalking(topNodes, null);
   }
 
-  private static class CollectAll implements SemanticNodeProcessor {
```

Comment:
Why are we touching this?

Reply:
This is not used anywhere. I assume it is costly to ensure unused classes works correctly compared with advantages to keep it.